README.md

```diff
@@ -271,7 +271,7 @@ This work is tracked in https://github.com/ros-tooling/cross_compile/issues/263.
 
 For a new user, this section walks you through a representative use case, step by step.
 
-This tutorial demonstrates how to cross-compile the [ROS 2 Demo Nodes](https://github.com/ros-tooling/demos) against ROS 2 Foxy, to run on an ARM64 Ubuntu system.
+This tutorial demonstrates how to cross-compile the [ROS 2 Demo Nodes](https://github.com/ros-tooling/demos) against ROS 2 Galactic, to run on an ARM64 Ubuntu system.
```

Comment:
Given that Foxy will outlive Galactic - I would prefer to leave the current LTS ROS 2 Distro as the example in the README. Would you mind reverting that portion of this?

Reply:
Sure all good.